### PR TITLE
[5.4] Added tests to make sure changing and renaming columns work

### PR DESCRIPTION
**Pull request by themsaid**

### CODE REVIEW

**Comment by taylorotwell** [comment]
Looks like maybe we're not requiring dev on travis or something?

**Comment by taylorotwell** [comment]
Says doctrine/dbal not installed.

File: composer.json [hunk 1]
```diff
@@ -76,6 +76,7 @@
         "pda/pheanstalk": "~3.0",
         "phpunit/phpunit": "~5.7",
         "predis/predis": "~1.0",
+        "doctrine/dbal": "~2.4",
```
Comment (GrahamCampbell) [inline composer.json]: Increase the min version so tests pass on all builds. :)

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by GrahamCampbell** [comment]
Might be an idea to bump the min version across the suggests in all composer files too. :)

**Comment by themsaid** [comment]
@GrahamCampbell I'm not sure what versions should be used 🤓

**Comment by GrahamCampbell** [comment]
> @GrahamCampbell I'm not sure what versions should be used 🤓

What you chose is fine. I'm saying update the other "suggest" blocks. :)

**Pull request merged**